log/experimental_level/level.go

```diff
@@ -67,42 +67,50 @@ func Debug(logger log.Logger) log.Logger {
 	return log.NewContext(logger).WithPrefix(levelKey, debugLevelValue)
 }
 
-// Config parameterizes the leveled logger.
-type Config struct {
-	// Allowed enumerates the accepted log levels. If a log event is encountered
-	// with a level key set to a value that isn't explicitly allowed, the event
-	// will be squelched, and ErrNotAllowed returned.
-	Allowed []string
+// New wraps the logger and implements level checking. See the commentary on the
+// Option functions for a detailed description of how to configure levels.
```

Comment:
Would you mind adding an extra sentence of commentary describing the default behavior with no options set?